parserasgeo/features/cross_section.py

```diff
@@ -514,6 +520,76 @@ def cut_line_ratio(self):
             last_pt = pt
 
         return cl_length/length
+    
+    def define_channel_n(self):
+        """
+        Checks if the bank stations are represented in Manning's n (i.e. if Manning's n is defined at the statiosn that correspond with the bank stations)
+        If they are not defined at the bank stations, add Manning's n values at those points.
+        """
+        
+        left_bank = self.bank_sta.left
+        right_bank = self.bank_sta.right
+        banks = [left_bank, right_bank]
+        
+        manning_tuples = self.mannings_n.values
+        manning_stations = [x[0] for x in manning_tuples]
+        
+        for bank in banks:
+            # if the bank station is not in the list, grab the Manning's n to the left
+            if bank not in manning_stations:
+                left_manning = [x[1] for x in manning_tuples if x[0] < bank]
```

Comment:
the "left_" prefix is used on several variables, but these variables appear to be used for both bank stations, is that correct? if so please drop the prefix.

Reply:
left in this case indicates the manning's n to the left of the bank station. this can, however, be confused with the left bank station. changed "left" to "leftward" to make the meaning clearer